### PR TITLE
Create fr.js

### PR DESCRIPTION
**Pull request by OpakoDev**
Some translations might be inaccurate but most of them are correct.
I also didn't edit the dates format as I didn't exactly know how can I edit them and t he "set times" field as I didn't see where does it go to notice the exact translation ^^

### CODE REVIEW

File: translations/fr.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+export default {
+  translation: {
+    'iRacing Week Planner': 'Planificateur hedbomadaire iRacing',
+    'Purchase guide': 'Guide d'achats',
```
Comment (remigarcia) [inline translations/fr.js:4]:
```suggestion
    'Purchase guide': 'Guide d\'achats',
```

File: translations/fr.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    'Set my tracks': 'Circuits possédés',
+    'Set my cars': 'Voitures possédées',
+    'Set favorite series': 'Séries favories',
```
Comment (remigarcia) [inline translations/fr.js:7]:
```suggestion
    'Set favorite series': 'Séries favorites',
```

File: translations/fr.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    'Licence': 'Licence',
+    'Official/Fixed': 'Officielle/Configuration fixe',
+    'Unofficial': 'Officieuse',
+    'Official': 'Officielle',
+    'Open setup': 'Configuration ouverte',
+    'Fixed setup': 'Configuration fixe',
+    'Content': 'Contenu',
+    'Owned cars only': 'Voitures possédées seulement',
+    'Owned tracks only': 'Circuits possédés seulement',
+    'Favorite series only': 'Séries favories seulement',
```
Comment (remigarcia) [inline translations/fr.js:30]:
```suggestion
    'Favorite series only': 'Séries favorites seulement',
```

File: translations/fr.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    'Favorite cars only': 'Voitures favories seulement',
```
Comment (remigarcia) [inline translations/fr.js:31]:
```suggestion
    'Favorite cars only': 'Voitures favorites seulement',
```

File: translations/fr.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    'Favorite tracks only': 'Circuits favoris seulement',
+    'Reset filters': 'Remettre les filtres à 0',
+    'Reset all settings': 'Remettre tous les paramètres à 0',
+
+    // Column Headers
+    'Columns': 'Colomnes',
```
Comment (remigarcia) [inline translations/fr.js:37]:
```suggestion
    'Columns': 'Colonnes',
```

File: translations/fr.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    'Race times': 'Durée de la course',
+    'Next race': 'Prochaine course',
+    'Season end': 'Fin de la saison',
+    'Link': 'Lien',
+    'Week': 'Semaine',
+    'Count': 'Nombre',
+
+    // Modals
+    'Use the checkbox to set the content you own, and use the star to set your favourite content.':
+      'Use the checkbox to set the content you own, and use the star to set your favourite content.',
```
Comment (remigarcia) [inline translations/fr.js:55]:
```suggestion
      'Utilisez les cases à cocher pour enregistrer le contenu que vous possédez et utilisez l\'étoile pour enregistrer vos contenus favoris.',
```

File: translations/fr.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    'Select all': 'Sélectionner tout',
+    'Select all oval': 'Sélectionner tous les ovaux',
```
Comment (remigarcia) [inline translations/fr.js:57]:
```suggestion
    'Select all oval': 'Sélectionner tous les ovales',
```

File: translations/fr.js [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    'Select all road': 'Sélectionner tous les routiers',
+    'Choose favorite series': 'Sélectionner les séries favories',
```
Comment (remigarcia) [inline translations/fr.js:59]:
```suggestion
    'Choose favorite series': 'Sélectionner les séries favorites',
```

File: translations/fr.js [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+    'Close': 'Fermer',
+    'Contributors': 'Contributeurs',
+    'Changelog': 'Journal des modifications',
+    'Purchase Guide': 'Guide d'achat',
```
Comment (remigarcia) [inline translations/fr.js:63]:
```suggestion
    'Purchase Guide': 'Guide d\'achat',
```

File: translations/fr.js [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+    'This tool was created by <1>tmoitie</1> (<3>Tom Moitié</3> on iRacing). Feel free to contact me via twitter or iRacing if you have any feedback or questions. The code is hosted publicly on <5>Github</5>. Thanks!':
+      'Cet outil a été créé par <1>tmoitie</1> (<3>Tom Moitié</3> sur iRacing). N'hésitez pas à me contacter sur Twitter ou iRacing si vous avez un retour à faire ou des questions à poser. Le code source est hébergé publiquement sur <5>Github</5>. Merci!',
```
Comment (remigarcia) [inline translations/fr.js:65]:
```suggestion
      'Cet outil a été créé par <1>tmoitie</1> (<3>Tom Moitié</3> sur iRacing). N\'hésitez pas à me contacter sur Twitter ou iRacing si vous avez un retour à faire ou des questions à poser. Le code source est hébergé publiquement sur <5>Github</5>. Merci !',
```

File: translations/fr.js [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    'Loading': 'Chargement',
+    'Cars for {{series}}': 'Véhicules pour {{series}}',
+    'Tracks for {{series}}': 'Circuits pour {{series}}',
+    'These unowned tracks from your favorite series appear multiple times for this season. You can purchase these tracks with the direct link.': 'Ces circuits non possédés apparaissent plusieurs fois dans vos séries favories cette saison. Vous pouvez les acheter via le lien direct.',
```
Comment (remigarcia) [inline translations/fr.js:69]:
```suggestion
    'These unowned tracks from your favorite series appear multiple times for this season. You can purchase these tracks with the direct link.': 'Ces circuits non possédés apparaissent plusieurs fois dans vos séries favorites cette saison. Vous pouvez les acheter via le lien direct.',
```

**Review by remigarcia — CHANGES_REQUESTED** [review]
Hello there :) I was starting to translate it when I saw this PR. I then allowed myself to review it.
Saw one untranslated message, some quotes errors and spell checks.

**Comment by tmoitie** [comment]
Thanks both! I'm merging what we've got here and left a comment filled with everything left to translate (stuff I've added since, sorry!). Merci beauçoup !

**Pull request merged**